### ISSUE TITLE
Hostname resolution capability

### ISSUE THREAD

**Issue by 459below** [fetzerms/cryptboot-ssh#20]
It seems - oddly enough - to be a tad non-trivial to enable hostnames for resolving the keyserver in initramfs. Furthermore it seems to work just fine, if it's using DHCP. I'm probably overlooking something here.

In any case, I'd deem it a useful capability. A proper hostname should make the setup more durable. ATM if the IP of the keyserver is changed, the scripts and the initrd need to be modified. I think if we have a hostname (and it's referenced in the `KnownHosts`) we can move the keyserver around without breaking unlock capability on every system.

**Comment by 459below** [comment]
This section should be applicable here:

https://wiki.gentoo.org/wiki/Custom_Initramfs#DNS

I haven't tried it yet, but it looks promising to me.

**Comment by fetzerms** [comment]
Yes, I read this too. But it surprises me a little. Using DHCP, it seems to work fine. This does not sound like a library problem to me. But we should really dig into this. Maybe adjusting/supplying a proper resolv.conf already works.
